Server/Models/ItemState.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Karma.Server.Models
+{
+    public enum ItemState
+    {
+        Ready, //Charity item is publicly seen and ready to be taken
+        Reserved, //Someone is interested in this item
+        Taken //This item has been collected and is no longer available
```

Comment:
Maybe it the states should be:
"Started" charity event
"InProgress" of collecting money 
"Finished",  
"Removed" the charity event was deleted and is marked for deletion(soft delete).
What do you think about this?

Reply:
Ok, rewrote it like this, just not sure about how well the ''charity'' and ''money collection'' parts go together.